rules/lfi-os-files.data

```diff
@@ -26,20 +27,28 @@
 .bash_
 .bashrc
 .boto
+.cache/
 .cache/notify-osd.log
 .cargo/
+.config.local.php.swp
```

Comment:
Why not only `.config.local.php`? It's matched using `pmFromFile` so it will work for both. Also, temporary file extensions are handled by other rule.